ui/app/routes/vault/cluster/settings/mount-secret-backend.js

```diff
@@ -1,16 +1,17 @@
 import Route from '@ember/routing/route';
-import UnloadModelRoute from 'vault/mixins/unload-model-route';
-import UnsavedModelRoute from 'vault/mixins/unsaved-model-route';
 import { inject as service } from '@ember/service';
 
-export default Route.extend(UnloadModelRoute, UnsavedModelRoute, {
-  store: service(),
-  // intentionally blank - we don't want a model until one is
-  // created via the form in the controller
-  model() {
-    return {};
-  },
-  activate() {
+export default class VaultClusterSettingsMountSecretBackendRoute extends Route {
```

Comment:
could your `@withConfirmLeave()` decorator be used here to preserve the functionality from the UnsavedModelRoute mixin?

Reply:
I saw some strange behavior with it, particularly with KV since there are sometimes expected dirty attributes on mount because there are two endpoints. I figured the mount screens were fairly small that it wasn't completely necessary to have the window.confirm when leaving